planning/behavior_velocity_planner/src/scene_module/detection_area/debug.cpp

```diff
@@ -20,13 +20,15 @@
 #ifdef ROS_DISTRO_GALACTIC
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 #else
+#include <std_msgs/msg/color_rgba.hpp>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #endif
 
```

Comment:
```suggestion
#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
#endif
#include <std_msgs/msg/color_rgba.hpp>

```
We should move it out to keep compatibility among ros2 versions.